compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -62,27 +62,14 @@ pub const METADATA_HEADER: &[u8] = &[b'r', b'u', b's', b't', 0, 0, 0, METADATA_V
 /// e.g. for `Lazy<[T]>`, this is the length (count of `T` values).
 trait LazyMeta {
     type Meta: Copy + 'static;
-
-    /// Returns the minimum encoded size.
-    // FIXME(eddyb) Give better estimates for certain types.
-    fn min_size(meta: Self::Meta) -> usize;
```

Comment:
I suppose I had hoped to be able to predict certain sizes well enough to have a lot of single-byte LEB128 encodings, but in retrospect it doesn't seem doable without a bunch of custom derives. In most cases, distances under 128 wouldn't make a difference anyway, and I suspect that's most of what `min_size` could ever handle (outside of `Lazy<[T]>`, I suppose).